applications/solvers/superDeliciousVanilla/superDeliciousVanilla.C

```diff
@@ -123,10 +136,21 @@ int main(int argc, char *argv[])
                 corrIter++;
             }
 
-            // Compute the continuity errors.
-            #include "computeDivergence.H"
+            // If starting without the t^(n-1) data, advance through the remaining
+            // outer iterations without doing anything by calling pimple.loop().
+            if (limitOuterLoop)
+            {
+                while (pimple.loop());
```

Comment:
Had to do a double-take on this code, but it makes sense to me that this will continue to call `pimple.loop()` until all the remaining requested outer iterations are complete.

Reply:
If you do not call pimple.loop() until all remaining iterations are complete, the iteration count will be off in the next time step, and the next time step behavior will not be correct.  I can't find a function to just reset instead of finishing out the loop.